config/initializers/statsd.rb

```diff
@@ -33,7 +33,21 @@ class << Samson
 end
 
 ActiveSupport::Notifications.subscribe("job_status.samson") do |*, payload|
-  Samson.statsd.increment "jobs.#{payload.fetch(:type)}.#{payload.fetch(:status)}"
+  tags = [
+    "project:#{payload.fetch(:project)}",
+    "stage:#{payload.fetch(:stage)}",
+  ]
+  total_latency = payload.fetch(:total_latency)
+  pr_production_latency = total_latency.fetch(:pr_production_latency)
+  staging_production_latency = total_latency.fetch(:staging_production_latency)
+
+  if pr_production_latency > 0
```

Comment:
`!= 0` so we see when the math goes wrong ?
... also 0 is a magic value, so better use a constant like "NOT_APPLICABLE" or so ?